autockt/envs/ngspice_ledro_d_fc.py

```
@@ -0,0 +1,365 @@
+        with open(LEDRO_D_FC.CIR_YAML, "r") as f:
+            yaml_data = yaml.load(f, OrderedDictYAMLLoader)
+
+        # design specs
+        if self.generalize == False:
+            specs = yaml_data["target_specs"]
+        else:
+            load_specs_path = (
+                LEDRO_D_FC.path + "/autockt/gen_specs/ngspice_specs_gen_ledro_d_fc"
+            )
+            with open(load_specs_path, "rb") as f:
+                specs = pickle.load(f)
```

Ruff (0.12.2)

`pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)